README.md

```diff
@@ -214,7 +214,40 @@ sudo emerge --ask awscli
 ```
 
 ### azure
-TBD (FIXME)
+`azure` uses `~/.azure/azureProfile.json`. This can be created using the `az` [command](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli):
+```
+$ az login`
+```
+It also requires that the environment variable `AZURE_AUTH_LOCATION` points to a JSON file. The JSON file will require a service provider active directory account to be created.
```

Comment:
Can mantle set this if it's unset so users do not need to?

Reply:
I'd say just pick one and go with it, needing to set an environment variable to run kola is a pain.

Reply:
@bgilbert: do you have a preference?

Reply:
One thing I'd add about setting this environment variable is that there isn't going to be a good way to Unset the environment variable. We tend to do platform specific configuration inside of the Cluster or API but those do not provide sufficient points for deferred cleanup of an environment variable.

Reply:
IIRC environment vars should die with the process.